packages/devextreme/js/ui/html_editor/matchers/wordLists.js

```diff
@@ -24,14 +24,15 @@ function removeNewLineChar(operations) {
 
 const getMatcher = (quill) => {
     const Delta = quill.import('delta');
+    const msStyleAttributeName = quill.LIST_STYLE_KEY;
 
     return (node, delta) => {
         const ops = delta.ops.slice();
 
         const insertOperation = ops[0];
         insertOperation.insert = insertOperation.insert.replace(/^\s+/, '');
         const listDecoratorMatches = insertOperation.insert.match(/^(\S+)\s+/);
-        const indent = listDecoratorMatches && getIndent(node);
+        const indent = listDecoratorMatches && getIndent(node, msStyleAttributeName);
```

Comment:
hm, cannot we just use `quill.LIST_STYLE_KEY` directly in `getIndent` to get rid of an additional variable and parameter passing?